### ISSUE TITLE
Error in   "rec_loss_lpips = self.first_stage_model.loss" in ddpm3d.py.Because in your config,    loss_config:  target: torch.nn.Identity 

### ISSUE THREAD

**Issue by songyang86** [mira-space/Mira#11]

**Comment by songyang86** [comment]
![image](https://github.com/user-attachments/assets/a8ac13ad-18bf-4072-bf38-89d7ffc38536)
